package.json

```
@@ -29,9 +29,13 @@
     "test": "npm run jest",
     "posttest": "aud --production",
     "test:ci": "npm run jest -- --ci --runInBand",
+    "test:examples": "npm run build && concurrently -c \"auto\" \"npm:test-example:*\"",
```

my experience with concurrently and alternatives is that it's too easy for failures to silently pass - i'd rather just run them explicitly and in serial.
```suggestion
    "pretest:examples": "npm run build",
    "test:examples": "concurrently -c \"auto\" \"npm:test-example:*\"",
```

---

That's fair.  I went with `concurrently` for the opposite reason.  If they were run serially, then it fails on the first failure, which may not expose the fact that one of the other is failing too (e.g. you fix one, only to find out the next one failed).  But I'm happy to switch that too.

---

Right - in theory concurrently is great. However, in practice, I've found that one of the parallel tasks can fail, without failing the overarching CI, allowing bugs to be merged, so i'd rather suffer the tradeoff of serial failures than silent ones.